### DISCUSSION TITLE
希望在发送页面能增加常用IP地址的选项

### DISCUSSION THREAD

**Discussion by oyster-zzz** [localsend/localsend#735] [L-简体中文]
目前发送页面无法自动检测到处于不同网段的设备，但手动输入IP地址可以向不同网段的接受设备进行发送。希望能在发送页面增加自定义的常用IP地址功能，避免每次向不同网段设备发送文件都需要手动输入其IP地址的流程

**Comment by KenChen1998** [comment]
希望可以保存常用的固定ip地址，这样可以方便隧道连接设备的传输。

**Comment by prowyh** [comment]
如果能对识别出的设备（Nearby devices）和手动输入的IP地址增加“分组管理”的功能，那就太棒啦！
